docs/openapi-docs/src/test/scalajvm-2/sttp/tapir/docs/openapi/VerifyYamlEnumeratumTest.scala

```diff
@@ -23,7 +23,8 @@ class VerifyYamlEnumeratumTest extends AnyFunSuite with Matchers {
     val actualYaml =
       OpenAPIDocsInterpreter()
         .toOpenAPI(List(endpoint.in("enum-test").out(jsonBody[FruitWithEnum])), Info("Fruits", "1.0"))
-        .toYaml
+        .openapi("3.1.0")
+        .toYaml3_1
```

Comment:
I think we should make 3.1.0 the default version in sttp-apispec, so that `.toYaml` gives you a 3.1 spec. This would mean renaming the current toYaml method to `.toYaml3_0`, and releasing a new minor version of apispec. Then the changes here will be bit smaller

Reply:
can you also check some of these generated yamls (not all of course, just a random sample) if they validate correctly in the swagger editor 5.0?

Reply:
I already checked few and they were ok

Reply:
ah ok, thanks :)